Sources/DangerDependenciesResolver/InlineDependenciesFinder.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+struct InlineDependenciesFinder {
+    let fileReader: FileReading
+    let config: ScriptManager.Config
+
+    init(fileReader: FileReading = FileReader(),
+         config: ScriptManager.Config) {
+        self.fileReader = fileReader
+        self.config = config
+    }
+
+    func resolveInlineDependencies(fromPath path: String) throws -> [InlineDependency] {
+        let lines = try fileReader.readText(atPath: path).components(separatedBy: .newlines)
+
+        var result = [InlineDependency]()
+
+        for line in lines {
+            if line.hasPrefix("import ") {
+                let components = line.components(separatedBy: config.dependencyPrefix)
+                guard components.count > 1,
+                    let lastComponent = components.last else {
+                    continue
+                }
+
+                let splittedImportString = lastComponent.trimmingCharacters(in: .whitespaces).components(separatedBy: " " + config.majorVersionPrefix)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerDependenciesResolver/InlineDependenciesFinder.swift;
  Line: 26;
-->

- :warning: Line should be 120 characters or less: currently 150 characters (`line_length`)